### ISSUE TITLE
Max_batches Question

### ISSUE THREAD

**Issue by Scovin** [pjreddie/darknet#1407]
Hi all,
I am running yolov3. I wonder what is the Max_batches?
setting:
Batch = 64
subdivisions=16
max_batches = 20000

![image](https://user-images.githubusercontent.com/33141724/51954587-bc492b80-247b-11e9-8704-bd2373f5d211.png)

Following this pic, the batch is 21.
So I have to wait the batch being 20000 and then it stops the training?

**Comment by nkalupahana** [comment]
That's right. The model also regularly exports backup weights files (backup directory in your `.data` file), though, so you can stop the training halfway and use those if you want to see if your training is working.
